test/cpp/interop/client_helper.cc

```diff
@@ -105,6 +105,13 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
     creds = FLAGS_custom_credentials_type == "google_default_credentials"
                 ? nullptr
                 : AccessTokenCredentials(GetOauth2AccessToken());
+  } else if (test_case == "pick_first_unary") {
+    ChannelArguments channel_args;
+    // force using pick first policy
```

Comment:
nit: this comment maybe confusing, suggest removing it or replacing with something like ("allow the LB policy to be configured with service config").

Reply:
+1